gradebookng/tool/src/java/org/sakaiproject/gradebookng/business/GradebookNgBusinessService.java

```diff
@@ -1615,6 +1655,43 @@ public String getAssignmentGradeComment(final String siteId, final long assignme
 		return null;
 	}
 
+	/**
+	 * Get the Excluded Status for a given student assignment grade
+	 *
+	 * @param assignmentId id of assignment
+	 * @param studentUuid uuid of student
+	 * @return the comment or null if none
+	 */
+	public Boolean getAssignmentExcludedStatus(final long assignmentId, final String studentUuid) {
+		final String siteId = getCurrentSiteId();
+		final Gradebook gradebook = getGradebook(siteId);
+		return gradebookService.getExculdedStatusForAssignmentByStudent(gradebook.getUid(), assignmentId, studentUuid);
+	}
+
+	/**
+	 * Set the Excluded Status for a given student assignment grade
+	 *
+	 * @param assignmentId id of assignment
+	 * @param studentUuid uuid of student
+	 */
+	public void setAssignmentExcludedStatus(final Long assignmentId, final String studentUuid) {
```

Comment:
As a set type method I would prefer to see this setter taking a value for it to set rather than leaving it up to the service.